pkg/sentry/devices/nvproxy/version.go

```diff
@@ -790,7 +822,20 @@ func Init() {
 			return abi
 		}
 
-		_ = addDriverABI(560, 35, 03, "f2932c92fadd43c5b2341be453fc4f73f0ad7185c26bb7a43fbde81ae29f1fe3", v555_42_02)
+		// The following exist on the "560" branch.
```

Comment:
560.28.03 exists on the main branch: https://github.com/NVIDIA/open-gpu-kernel-modules/commit/448d5cc65624d3aa69015efa0d3fb50fd9729f41.

You can make the comment "560.28.03 is an intermediate unqualified version."
It should inherit from v555_42_02, which you have implemented correctly.